UKHomeOffice/cerberus-service
# COP-10462 - Task details itinerary

## Description
This PR adds the itinerary section on the task details page.

## To Test
- Pull and run against dev
- In `src/routes/airpax/TaskDetails/TaskDetailsPage.jsx` file. 
- On `line 35`, replace the content of data with the contents in fixture file `taskData_AirPax_TaskDetails_Versions.fixture.json`.
- Locally, load the following URL http://localhost:8080/airpax/tasks/DEV-20220419-001 and confirm that the itinerary section resembles that of the image below.

![image](https://user-images.githubusercontent.com/19333750/167375012-5656741e-e58c-4134-a778-89f34503a200.png)

## Developer Checklist

\* Required
